plugins/readme/api_key.go

```diff
@@ -0,0 +1,87 @@
+package readme
+
+import (
+	"context"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func APIKey() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.APIKey,
+		DocsURL:       sdk.URL("https://docs.readme.com/main/reference/intro/authentication"),
+		ManagementURL: sdk.URL("https://docs.readme.com/main/reference/intro/authentication"),
+		Fields: []schema.CredentialField{
+			{
+				Name:                fieldname.Username,
+				MarkdownDescription: "Currently logged-in ReadMe user.",
```

Comment:
We show this text _before_ the user is logged in. I think we should slightly reword to incorporate this. Similar to [how we do for MySQL username field](https://github.com/1Password/shell-plugins/blob/main/plugins/mysql/database_credentials.go#L32)

```suggestion
				MarkdownDescription: "ReadMe user to log-in as.",
```

Reply:
Thanks for the clarification, made that change in https://github.com/1Password/shell-plugins/pull/106/commits/04ac3272820eb806de4b1beb50c946dae77b71ae (with a slight grammar tweak to your suggestion)